UIKitHomework/UIKitHomework/NewLabelViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ViewController.swift
+//  UIKitHomework
+//
+//  Created by Polina on 21.06.2021.
+//
+
+import UIKit
+
+final class NewLabelViewController: UIViewController {
+    
+    public var yourLabel = UILabel()
```

Comment:
```suggestion
   var yourLabel = UILabel()
```
public мы в данном случае не используем, достаточно просто internal (который мы имеем по умолчанию).

Reply:
поправила